code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1579,6 +1579,7 @@
 	name = "Invisible Powder"
 	colorname = "invisible"
 	color = "#FFFFFF00" // white + no alpha
+	random_color_list = list("#FFFFFF") //because using the powder color turns things invisible
```

Comment:
Won't just make you white? shouldn't you set the alpha to 0 here?

Reply:
Setting the alpha to 00 makes you permanently invisible. This is the solution that most closely resembles the previous behavior.